docs/source/markdown/options/exec.md

```diff
@@ -0,0 +1,18 @@
+####> This option file is used in:
+####>   podman podman-container.unit.5.md.in
+####> If file is edited, make sure the changes
+####> are applicable to all of those.
+### `Exec=command`
+
+Additional arguments for the container; this has exactly the same effect as passing
+more arguments after a `podman run <image> <arguments>` invocation.
+
+The format is the same as for [systemd command lines](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines),
+However, unlike the usage scenario for similarly-named systemd `ExecStart=` verb
```

Comment:
```suggestion
However, unlike the usage scenario for similarly-named systemd `ExecStart=` verbs
```

Reply:
Shouldn't it be a singular? There's just single `ExecStart` verb. If it should really be a plural, I will also change the `operates` to `operate` in the next line, ok?

Reply:
Agree singular is more appropriate